src/main/scala/interfaces/Afe.scala

```diff
@@ -195,4 +202,12 @@ class MainbandAfeIo(
     * @group clock
     */
   val txClockPark = Output(Bool())
+
+  /** Mainband PLL Lock.
+   *
+   * Indicates whether the mainband clock is stable.
+   *
+   * @group clock
+   */
+  val mbPllLock = Output(Bool())
```

Comment:
can just call this `pllLock`, since the "mainband" part is implied by this signal being part of `MainbandAfeIo`